ci/cpu/prebuild.sh

```diff
@@ -15,8 +15,10 @@ else
 fi
 
 #We only want to upload libcudf_kafka once per python/CUDA combo
-if [[ "$PYTHON" == "3.7" -a "$CUDA" == "10.1"]]; then
-    export UPLOAD_KAFKA=1
+if [[ "$PYTHON" == "3.7" ]]; then
+    if [[ "$CUDA" == "10.1" ]]; then
+        export UPLOAD_KAFKA=1
+    fi
 else
     export UPLOAD_KAFKA=0
 fi
```

Comment:
If Python is 3.7 and CUDA is 10.2 `UPLOAD_KAFKA` will be undefined

Reply:
With double brackets you can use `&&` for `and` instead of `-a`

Reply:
Ohh I knew that. Stupid overlook on my part.